app/decorators/user_decorator.rb

```diff
@@ -1,4 +1,5 @@
 include ActionView::Helpers::DateHelper
+include ActionView::Helpers::UrlHelper
```

Comment:
these includes should go inside the class definition right? So they don't leak into the global namespace?